### PR TITLE
Sources Backup blogpost

### PR DESCRIPTION
**Pull request by czoido**

### CODE REVIEW

**Review by RubenRBS — APPROVED** [review]
10/10

File: _posts/2023-09-29-backup-sources-feature.markdown [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+comments: false
+title: "Enhancing Build Reliability with Conan's Source Backup Feature"
```
Comment (memsharded) [inline _posts/2023-09-29-backup-sources-feature.markdown:4]: Maybe adding "third-party source backup feature"?

**Review by memsharded — APPROVED** [review]
This is fantastic

File: _posts/2023-09-29-backup-sources-feature.markdown (hunk 1 above)
Comment (czoido) [reply _posts/2023-09-29-backup-sources-feature.markdown:4]: Yes, that's better 👍

Comment (czoido) [inline _posts/2023-09-29-backup-sources-feature.markdown:4]:
```suggestion
title: "Enhancing Build Reliability with Conan's third-party source backup feature"
```

File: _posts/2023-09-29-backup-sources-feature.markdown [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+Thanks to this feature, you can configure Conan so that when a recipe downloads the
+sources of a library, it allows you to fetch sources from a file server within your
+infrastructure. This is done in a completely transparent manner for the developer and can
+be set up to be used only as a last resort in case the URL of the original sources fails.
+
+Some of the advantages of using this feature are:
+
+- **Reproducibility**: Ensures long-term reproducibility in your builds.
+- **Traceability**: Keeps a record of the origin of downloaded files.
+- **Security**: Stores the sources in your own infrastructure, reducing reliance on
```
Comment (RubenRBS) [inline _posts/2023-09-29-backup-sources-feature.markdown:26]: Isn't this robustness instead of security?

Comment (czoido) [reply _posts/2023-09-29-backup-sources-feature.markdown:26]: Yes, robustness fits well, let's change that
```suggestion
- **Robustness**: Stores the sources in your own infrastructure, reducing reliance on
```

**Pull request merged**